src/pages/dashboard/_StateCumulativeTestsContainer.js

```diff
@@ -78,14 +105,37 @@ function groupAndSortStateDaily(query) {
     })
     .entries(data)
 
-  return grouped.sort((a, b) => {
-    const lastA = a.values[0]
-    const lastB = b.values[0]
+  const groupedPerCapita = grouped.map(stateData => {
+    const population = statePopulations[stateData.key]
+      ? statePopulations[stateData.key]
+      : territoryPopulations[stateData.key]
+    const clonedData = cloneDeep(stateData)
 
-    const lastATotal = calculateTotal(lastA)
-    const lastBTotal = calculateTotal(lastB)
-    return lastBTotal - lastATotal
+    clonedData.values = clonedData.values.map(value => {
+      const clonedValue = cloneDeep(value)
+
+      // divide by population to determine per capita percentages
+      clonedValue.positive /= population
+      clonedValue.negative /= population
+
+      return clonedValue
+    })
+
+    return clonedData
   })
+
+  return {
+    totals: sortGroupedData(grouped),
+    perCapita: sortGroupedData(groupedPerCapita),
+  }
+}
+
+function normalizeByStatePopulation(val, stateKey) {
```

Comment:
this doesn't appear to be called anywhere

Reply:
Good catch, fixed in 33658fd.